lib/capybara/spec/session/node_spec.rb

```diff
@@ -102,6 +102,12 @@
       expect { @session.first('//textarea[@readonly]').set('changed') }.to raise_error(Capybara::ReadOnlyElementError)
     end
 
+    it 'should use global default options', requires: [:js] do
```

Comment:
test needs to reset the setting at the end, or add it to https://github.com/teamcapybara/capybara/blob/master/lib/capybara/spec/spec_helper.rb#L21

Reply:
This was fixed